starter-code/public/scripts/models/repo.js

```diff
@@ -7,11 +7,23 @@ var app = app || {};
   repos.all = [];
 
   repos.requestRepos = function(callback) {
-    // TODO: How would you like to fetch your repos? Don't forget to call the callback.
+
+    $('#about').show().siblings().hide(); // REVIEW: We have a slight refactor in selectors here, which has reduced the amount of code from the last lab.
```

Comment:
This is already handled in `aboutController`. jQuery DOM manipulation should never happen in the model, only views or controllers.